backend/sep20.js

```diff
@@ -0,0 +1,37 @@
+const axios = require("axios");
+
+const lookup = async function(nodeId) {
+  let accountResp;
+  try {
+    accountResp = await axios.get(
+      `https://horizon.stellar.org/accounts/${nodeId}`
+    );
+  } catch (e) {
+    // This key doens't exist on the ledger, it doesn't support SEP20
+    return null;
+  }
+  const homeDomain = accountResp.data.home_domain;
+  // This key doesn't set a homeDomain option, it doesn't support sep20
+  if (!homeDomain) return null;
+
+  const tomlURL = `http://${homeDomain}/.well-known/stellar.toml`;
+  const tomlResp = await axios.get(tomlURL);
+  const toml = tomlResp.data;
+  const lines = toml.split("\n");
+  // Check to ensure the toml matches the public key
+  let verified = false;
+  lines.forEach(line => {
```

Comment:
Should we consider using a [TOML parsing library](https://www.npmjs.com/package/toml) here? There are a few variants of the `KEY="String Value"` format, such as `KEY = "String Value"` (with an arbitrary number of whitespaces) or `KEY='''value'''` which are valid TOML [see specs [here](https://github.com/toml-lang/toml)] but would trigger a false negative here.

Reply:
good call, much more readable too